test/support/pages/Common.js

```diff
@@ -31,7 +31,10 @@ define(function (require) {
             if (testStatusPage !== false) {
               return self.checkForKibanaApp()
               .then(function (kibanaLoaded) {
-                if (!kibanaLoaded) throw new Error('Kibana is not loaded, retrying');
+                if (!kibanaLoaded) {
+                  throw new Error(moment().format('HH:mm:ss.SSS') + ': ' +
```

Comment:
Is this just replicating the `Common.log` syntax? Can you just use that method and *also* throw the error?